spec/lib/healthcheck/government_data_check_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.describe Healthcheck::GovernmentDataCheck do
+  describe "#status" do
+    context "when government data is unavailable" do
+      it "returns critical" do
+        gov_data_check = described_class.new
```

Comment:
I'd wonder if doing this a bunch is worth the 5 characters we save?

Reply:
I've changed this file about a bit and moved this to a `let` block at the top. It's cleaned the file up some especially given the new tests I introduced.